ring/client/pool.go

```diff
@@ -173,9 +175,18 @@ func (p *Pool) removeStaleClients() {
 	}
 }
 
-// cleanUnhealthy loops through all servers and deletes any that fails a healthcheck.
+// cleanUnhealthy loops through all servers and deletes any that fails a healthcheck
```

Comment:
The change here was to remove a full stop?
Seems unnecessary, though you could s/fails/fail/.